pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -436,3 +441,20 @@ class PagerState(
         )
     }
 }
+
+/**
+ * A flow which emits the [PagerState.currentPage] as it changes due after completed scrolls.
+ *
+ * This flow is not meant to be used for updating any UI within the attached [HorizontalPager]
+ * or [VerticalPager]. For that use-case, you should read
+ * [PagerScope.currentPage] and [PagerScope.currentPageOffset] from the content scope.
+ *
+ * @sample com.google.accompanist.sample.pager.PageChangesSample
+ */
+@ExperimentalPagerApi
+inline val PagerState.pageChanges: Flow<Int>
+    get() = snapshotFlow { isScrollInProgress }
+        // Only emit when the scroll has finished
+        .filter { isScrollInProgress -> !isScrollInProgress }
```

Comment:
would `filterNot { it } // only emit…` be clearer?

Reply:
Ah good call, always forget about `filterNot()`